mopro-ffi/tests/bindings/test_mopro_gen_benchmarks_report.swift

```diff
@@ -0,0 +1,26 @@
+import mopro
+import Foundation
+
+let path = FileManager.default.currentDirectoryPath + "/../../../../mopro-core/benchmarks/gpu_explorations/msm_bench_swift_laptop.csv"
+let fileURL = URL(fileURLWithPath: path)
+var fileContent = "num_msm,avg_processing_time(ms),total_processing_time(ms)\n"
+
+// generate trials = [1, 500, 1000, 1500, ... ,10000]
+let trials: [UInt32] = (0..<21).map { max($0 * 500, 1)}
+
+for each in trials {
+    do {
+        // for tracking the progress
+        print("Running benchmark with \(each) MSMs...")
+        let benchData: BenchmarkResult = try runMsmBenchmark(numMsm: each)
```

Comment:
So for 10k MSMs it will do all these in Rust and only return results once done, right? Just trying to understand if it is going back and forth between swift and rust or something